packages/radix-ui-themes/src/components/spinner.tsx

```diff
@@ -44,10 +44,16 @@ const Spinner = React.forwardRef<SpinnerElement, SpinnerProps>((props, forwarded
       <span>
         {/**
          * `display: contents` removes the content from the accessibility tree in some browsers,
-         * so we force remove it with `aria-hidden` and re-add it in the tree with `VisuallyHidden`
+         * so we force remove it with `aria-hidden`
          */}
-        <span style={{ display: 'contents', visibility: 'hidden' }}>{children}</span>
-        <VisuallyHidden>{children}</VisuallyHidden>
```

Comment:
I've removed this, as by adding `inert`, the element also is removed from the accessibility tree anyway.